biosimulations/apps/dispatch-service/src/app/services/sbatch/sbatch.service.ts

```diff
@@ -29,7 +29,7 @@ module load singularity/3.1.1
 export XDG_RUNTIME_DIR=${homeDir}/singularityXDG/
 date
 \`wget ${apiDomain}run/${simId}/download -O "${tempSimDir}/in/project.omex" 1>"${tempSimDir}/out/job.output" 2>&1\`
-command=\\" singularity pull ${simulator} && singularity run -B ${tempSimDir}/in:/root/in -B ${tempSimDir}/out:/root/out ${simulator} -i '/root/in/project.omex' -o /root/out\\"
+command=\\" singularity pull --force ${simulator} && singularity run -B ${tempSimDir}/in:/root/in -B ${tempSimDir}/out:/root/out ${simulator} -i '/root/in/project.omex' -o /root/out\\"
```

Comment:
This is what I am currently trying. 
I just need to confirm that this wont cause the cache to be overwritten and have all the layers downloaded again.